Tribler/Core/Modules/restapi/downloads_endpoint.py

```diff
@@ -94,11 +96,12 @@ def render_GET(self, request):
                              "speed_down": download.get_current_speed(DOWNLOAD),
                              "speed_up": download.get_current_speed(UPLOAD),
                              "status": dlstatus_strings[download.get_status()],
-                             "size": download.get_length(), "eta": download.network_calc_eta(),
+                             "size": download.get_def().get_length(), "eta": download.network_calc_eta(),
```

Comment:
@lfdversluis I just noticed that in subsequent lines, I sometimes have multiple (short) items on the same line so I would rather leave it like this to be consistent.